solr/core/src/java/org/apache/solr/util/plugin/NamedListInitializedPlugin.java

```diff
@@ -25,5 +25,10 @@
  * @since solr 1.3
  */
 public interface NamedListInitializedPlugin {
-  void init( @SuppressWarnings({"rawtypes"})NamedList args );
+  /**
+   * <code>init</code> will be called just once, immediately after creation.
```

Comment:
Nice to see you adding some docs here.  Let's also state that "args" is never null?